include/work_queue.h

```diff
@@ -1,13 +1,6 @@
-//
-// Created by victor on 6/2/21.
-//
-
-#ifndef TEST_WORK_QUEUE_H
-#define TEST_WORK_QUEUE_H
-
+#pragma once
 #include <buffer_tree.h>
-#include "supernode.h"
-// TODO: switch references from Node to node_t in types, change CMake flags
+// TODO: switch references from Node to node_t in types
```

Comment:
Can we get rid of this?

Reply:
we haven't done it though. there's still `Node` type in the buffer tree repo that we use

Reply:
Guess so